internal/release/main.go

```diff
@@ -119,3 +121,27 @@ func generateFile(name, version string) error {
 	jsonEncoder.SetIndent("", "  ")
 	return jsonEncoder.Encode(downloadArchive)
 }
+
+func Builder() *cobra.Command {
+	opts := Opts{}
+	cmd := &cobra.Command{
+		Use:   "main",
+		Short: "Generate the download center json file",
+		Example: `
+  Generate the download center json file for mongocli
+  $ main --version 1.23.0`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			fmt.Printf("Generating JSON: %s\n", opts.fileName)
+			return generateFile(opts.fileName, opts.version)
+		},
+		PostRun: func(cmd *cobra.Command, args []string) {
+			fmt.Printf("File %s has been generated\n", opts.fileName)
+		},
+	}
+
+	cmd.Flags().StringVar(&opts.version, flag.Version, "", "release version.")
+	cmd.Flags().StringVar(&opts.fileName, flag.File, "mongocli.json", "file name of the download center json file.")
+
+	_ = cmd.MarkFlagFilename(flag.Version)
```

Comment:
[nit] 
```suggestion
	_ = cmd.MarkFlagFilename(flag.File)
```

Reply:
the file flag is not mandatory as I have set a default value to mongocli.json. Are you suggesting removing the default value and making it mandatory?

Reply:
sorry `MarkFlagFilename` doesn't mark things as mandatory `MarkFlagRequired` does

Reply:
ops. I was sure I was using `MarkFlagRequired`. Thanks!

Reply:
Updated